celest/coordinates/frames/itrs.py

```diff
@@ -0,0 +1,103 @@
+
+
+from celest.coordinates.frames.base_positions import Position3d
+from celest.file_save import TextFileWriter
+from celest.units.core import Unit
+from celest.units.quantity import Quantity
+from celest import units as u
+import numpy as np
+
+
+class ITRS(Position3d):
```

Comment:
Is it worth creating a `Cartesian3d` class that inherits the `Position3d` class and is inherited by both the `ITRS` and `GCRS` classes?

Reply:
Not at this time...